docs/specialized_actions.md

```diff
@@ -62,9 +60,11 @@ Rails.application.routes.draw do
 end
 ```
 
-## Customizing Default Sorting
+## Specialized ordering
 
-To set the default sorting on the index action you could override `default_sorting_attribute` or `default_sorting_direction` in your dashboard controller like this:
+You can order records on index actions
```

Comment:
Ordering can be decided by the user in many cases by clicking the table headers. This piece of advice refers to the default order:

```suggestion
You can set the default order of records on index actions
```